app/serializers/tournament_serializer.rb

```diff
@@ -24,16 +24,19 @@
 #  fk_rails_...  (organiser_id => users.id)
 #
 
-class TournamentSerializer < ActiveModel::Serializer
-  attributes :competitors_limit,
-             :description,
-             :id,
-             :name,
-             :organiser_id,
-             :result_names,
-             :starts_at,
-             :status
+class TournamentSerializer < Blueprinter::Base
+  identifier :id
 
-  has_many :competitors
-  has_many :rounds
+  fields :competitors_limit,
+         :description,
+         :name,
+         :organiser_id,
+         :result_names,
+         :starts_at,
+         :status
+
+  view :extended do
+    association :competitors, blueprint: CompetitorSerializer
+    association :rounds, blueprint: RoundSerializer
```

Comment:
Coming over from https://github.com/procore/blueprinter/issues/161,  you'll need to explicitly pass in `view: :extended`
```ruby
association :rounds, blueprint: RoundSerializer, view: :extended`
```

Reply:
@philipqnguyen, it works, thanks! I think it would be a good idea to add this information to the Blueprinter README. Right now the `view` parameter is only mentioned in the context of the `.render` method.

Reply:
Good call. I'll update the readme in a bit!